### DISCUSSION TITLE
Extend the list of supported languages in AIR

### DISCUSSION THREAD

**Discussion by llorenzo** [airsdk/Adobe-Runtime-Support#605] [Q&A]
https://help.adobe.com/en_US/air/build/WSB2927578-20D8-4065-99F3-00ACE6511EEE.html

The AppStore and PlayStore support many more languages.

**Comment by zwetan** [comment]
I think those are mainly related to desktop AIR installer

for the app itself you have the [flash.globalization](https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/globalization/package-detail.html) package

also see [Choosing a locale](https://help.adobe.com/en_US/as3/dev/WS9b644acd4ebe59993a5b57f812214f2074b-8000.html#WS9b644acd4ebe59993a5b57f812214f2074b-7ffd)
